recipes/cpp-jwt/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
-set(CMAKE_VERBOSE_MAKEFILE TRUE)
-
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+find_package(cpp-jwt REQUIRED)
```

Comment:
It can't work without `cmake_find_package` or `cmake_find_package_multi` generators in conanfile.py of test_package.